bin/api-docs/update-api-docs.js

```diff
@@ -63,9 +63,9 @@ const TOKEN_PATTERN = /<!-- START TOKEN\((.+?(?:\|(.+?))?)\) -->/g;
 /**
  * Given an absolute file path, returns the package name.
  *
- * @param {string} file Absolute path.
+ * @param  {string} file Absolute path.
  *
- * @return {string} Package name.
+ * @return {string}      Package name.
```

Comment:
Maybe I'm missing something, but I don't get why we have to add these spaces here?

Reply:
Sounds like the rule aligns both `@param` _and_ `@return` lines, not just `@param` as the coding standards suggest. Although the docs are not super clear about it, tbh. But in WP core the `@return` lines aren't usually aligned.

I think the rule would need to be made configurable to allow for this.

Reply:
Right, it would be great to exclude `@return` from this formatting 👍🏻